src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import { User, UserInfo } from '../UserInfo/UserInfo';
+
+export interface Todo {
+  userId: number,
+  id: number,
+  title: string,
+  completed: boolean,
+  user?: User | null,
+}
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { user, title, completed } = todo;
+
+  return (
+
+    <article
+      data-id={todo.id}
+      className={`TodoInfo${completed ? ' TodoInfo--completed' : ''}`}
```

Comment:
use the following tool here, using ternary with empty string is not a solution
https://www.npmjs.com/package/classnames